packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2075,19 +2075,20 @@ export const components = ({
           maxWidth: `calc(${odysseyTokens.TypographyLineLengthMax} / 2)`,
           minWidth: "unset",
           minHeight: "unset",
-          padding: `${odysseyTokens.Spacing4} 0`,
+          padding: `${odysseyTokens.Spacing4} ${odysseyTokens.Spacing1}`,
+          fontSize: odysseyTokens.TypographySizeHeading6,
           fontFamily: odysseyTokens.TypographyFamilyHeading,
           lineHeight: odysseyTokens.TypographyLineHeightUi,
           overflow: "visible",
 
           ...(ownerState.textColor === "inherit" && {
-            color: "inherit",
+            color: odysseyTokens.HueNeutral600,
```

Comment:
If we are explicitly setting this color, rather than inheriting the currentColor we probably don't need this prop anymore.

Reply:
I see it's also setting `opacity: 1`. Not sure why that is needed. Maybe some weird MUI thing? @jordankoschei-okta Any thoughts or context on this `inherit` prop?

Reply:
good point, I'll update that.

Reply:
> I see it's also setting `opacity: 1`. Not sure why that is needed. Maybe some weird MUI thing? @jordankoschei-okta Any thoughts or context on this `inherit` prop?

I looked into this and saw that without this override, the default `opacity` is `.6` for the unselected tabs, which is the default MUI behavior. I think we'll need to keep it because the contrast is too low without it (2.38:1 vs 5.09:1 for `opacity: 1`).